tools/homarr-dev/internal/docker/api.go

```
@@ -0,0 +1,187 @@
+		return nil, fmt.Errorf("contact Docker engine: %w", err)
+	}
+	if response.StatusCode < 200 || response.StatusCode > 299 {
+		body, _ := io.ReadAll(io.LimitReader(response.Body, 4096))
+		_ = response.Body.Close()
+		return nil, fmt.Errorf("pull %s: %s", image, engineErrorMessage(response.Status, body))
+	}
+
+	messages := make(chan PullMessage, 64)
+	go func() {
+		defer close(messages)
+		defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)